aws/eip.go

```diff
@@ -0,0 +1,94 @@
+package aws
+
+import (
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func getOrSetFirstSeenTag(svc *ec2.EC2, address ec2.Address) (time.Time, error) {
+	key := "cloud-nuke-first-seen"
+	layout := "2006-01-02 15:04:05"
```

Comment:
These should probably be `const` outside of this method.

Reply:
I'll make them accept these values as arguments instead so the constants aren't global to the package